RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -1111,20 +1111,37 @@ boolean __restore(string resource_type, int goal, int meat_reserve, boolean useF
 
   boolean use_opportunity_blood_skills(int hp_restored_per_use){
     boolean success = true;
```

Comment:
Nit: move `success` down to above the loop, replace intermediate `return success` with `return true`